DataFormats/Math/interface/ExtVec.h

```diff
@@ -3,21 +3,47 @@
 
 #include<type_traits>
 
-typedef float  __attribute__( ( vector_size(  8 ) ) ) float32x2_t;
-typedef float  __attribute__( ( vector_size( 16 ) ) ) float32x4_t;
-typedef float  __attribute__( ( vector_size( 32 ) ) ) float32x8_t;
-typedef double __attribute__( ( vector_size( 16 ) ) ) float64x2_t;
-typedef double __attribute__( ( vector_size( 32 ) ) ) float64x4_t;
-typedef double __attribute__( ( vector_size( 64 ) ) ) float64x8_t;
+#ifdef __clang__
```

Comment:
how did we get to having to do fixes here.
I recall we switched to SSE in #6611
So, how did clang see an issue here?